lib/semian/sysv_integer.rb

```diff
@@ -0,0 +1,12 @@
+module Semian
+  module SysV
+    class Integer < Semian::Simple::Integer #:nodoc:
```

Comment:
I think this is a too unexpected way to provide the fallback. The fallback shouldn't be the SysV classes changing their implementation with `super` and inheritance, it should be done at boot-time. This is dangerous and would be incredibly difficult to debug.